### DISCUSSION TITLE
Using require instead of import

### DISCUSSION THREAD

**Discussion by mario-lemes** [idiotWu/smooth-scrollbar#356] [Q&A] [answered]
Hey @idiotWu 

Great job in this component!

I'm wondering if it's possible use 'require' in order to import the component instead of 'import'. If is that possible, How is suppose to be?

Thank you.

**Comment by dolphin-wood** [comment, marked as answer]
Sure that's possible! Since this module is defined as UMD, you can simply use `var Scrollbar = require('smooth-scrollbar')` in your code.
